docs/comunidad/charlas-y-presentaciones.md

```diff
@@ -0,0 +1,127 @@
+---
+id: charlas-y-presentaciones
+title: Charlas y Presentaciones de nuestro Equipo
+sidebar_label: Charlas y Presentaciones
+description: Charlas y presentaciones impartidas por miembros de nuestro equipo.
+keywords: [Charlas, Blockchain, Presentaciones, EOSIO, EOS]
+---
+
+# **Presentaciones y vídeos**
+
+En esta sección encontrarás presentaciones y vídeos de charlas impartidas por nosotros.
+
+___
+
+![EOS Costa Rica pre practica](/img/charlas/Pre-practica.webp)
+
+
+
+
+## Tema: EOSCR Pre-practica UI/UX
+
+**Por:** Jorge Murillo.
+
+**Idioma:** Español.
+
+**Fecha:** 4 de Febrero, 2021.
+
+### Grabación de la charla:
+
+**[EOS Pre-Practica UI/UX](https://us02web.zoom.us/rec/share/WWDqs5rB2r5rEdulNKUbOHcTBfiw6_ZTII8fB-UBjabJrQ3wTBvndkvnpWJupeBL.2n3yzt_rczzmKzj3)**
+
+
+
+___
+
+![EOS Costa Rica Tech Stack Review](/img/charlas/eos-cr.webp)
+
+
```

Comment:
Adding more than two blank lines does not increase spacing, i would remove blank lines to keep code more compact








text

Reply:
for example there are way too many blank lines here 

<img width="238" alt="image" src="https://user-images.githubusercontent.com/5632966/181663148-5e416327-5029-41df-91b8-167f423c3e3a.png">